warehouse/manage/views.py

```diff
@@ -337,7 +339,16 @@ def _key(project):
     effective_principals=Authenticated,
 )
 def manage_project_settings(project, request):
-    return {"project": project}
+    n_releases = (
```

Comment:
nit: `release_count` might be a more human friendly and obvious name than `n_releases`